AbSuppliers.php

```diff
@@ -287,6 +287,42 @@ public function displaySupplierPartners($atts)
         return $response;
     }
 
+    public function displaySupplierPartnersForEnergy($atts)
+    {
+        $atts['partners_only'] = true;
+
+        $atts = $this->processMultipleProductCats($atts);
+
+        list($atts, $supplierLogos) = $this->preparedSuppliersLogoData($atts);
+
+        $breakPoint = $atts['mod'];
+        $counter    = 0;
+        $response   = '<div class="row">';
+
+        foreach ($supplierLogos as $supplier) {
```

Comment:
Kan je hier niet gewoon foreach ($supplierLogos as $index => $supplier) van maken, zodat counter niet nodig is?